build/builder.go

```diff
@@ -249,9 +248,9 @@ type Builder struct {
 	todo         []*zoekt.Document
 	size         int
 
-	parserMap ctags.ParserMap
-
-	building sync.WaitGroup
+	parserMap  ctags.ParserMap
+	docChecker zoekt.DocChecker
+	building   sync.WaitGroup
```

Comment:
minor: I would group this with the "todo" field section since it has to do with stuff we do while building up todo. minor convention in go to group fields with respect to concurrency. This living with building can be seen as misleading.